librz/signature/sign.c

```diff
@@ -2,6 +2,6 @@
 // SPDX-FileCopyrightText: 2021 deroad <wargio@libero.it>
 // SPDX-License-Identifier: LGPL-3.0-only
 
-#include <rz_sign.h>
+#include <rz_flirt.h>
 
-RZ_LIB_VERSION(rz_sign);
+RZ_LIB_VERSION(rz_flirt);
```

Comment:
I do not agree with this change - while the underlying technology of our signatures is FLIRT, the API and module name should remain generic. FLIRT is a relatively old technology, maybe at some point, we will decide to implement something newer and better. Thus, the name of the module and most of the API should remain the same - `rz_sign`.

Reply:
TBH at this point the api does not look too generic. see https://github.com/rizinorg/rizin/blob/dfdc45156550f4532beac89ad2319419ab14dd6c/librz/include/rz_flirt.h, RzFlirtCompressedOptions, rz_sign_flirt_parse_header_compressed_pattern_from_buffer, rz_sign_flirt_parse_string_pattern_from_buffer, rz_sign_flirt_apply, rz_sign_flirt_node_count_nodes, RzFlirtInfo, etc. etc.

Reply:
ok, i have decided to revert it back to `rz_sign` since the lib is called `rz_sign`.

Reply:
Thank you.